.env

```diff
@@ -14,3 +14,10 @@ SPRINGBOARD_BUCKET=mbta-ctd-dataplatform-dev-springboard
 ARCHIVE_BUCKET=mbta-ctd-dataplatform-dev-archive
 ERROR_BUCKET=mbta-ctd-dataplatform-dev-error
 INCOMING_BUCKET=mbta-ctd-dataplatform-dev-incoming
+# mbta-performance with personal access
+PUBLIC_BUCKET=mbta-ctd-dataplatform-dev-archive 
```

Comment:
The LAMP team doesn't have access to the `mbta-performance` bucket, but this should be changed to that bucket if/when the LAMP team does get access

Reply:
we already use PUBLIC_ARCHIVE_BUCKET in the performance manager for archiving data for transit matters. can we set this to be the same var?

Reply:
Yup, didn't see that env var previously.
Incorporated